### PR TITLE
Disable debug symbols on circle CI

### PR DESCRIPTION
**Pull request by Luthaf**
As discussed in last meeting, this might help with OOM errors on circle CI.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,7 +45,7 @@ jobs:
     environment:
       CC: gcc-9
       CXX: g++-9
-      CMAKE_EXTRA: -DCMAKE_BUILD_TYPE=Debug
+      CMAKE_EXTRA: -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS_DEBUG=
```
Comment (agoscinski) [inline .circleci/config.yml]: why is this needed `DCMAKE_CXX_FLAGS_DEBUG=`?

Comment (Luthaf) [reply .circleci/config.yml]: This set `CMAKE_CXX_FLAGS_DEBUG` to the empty string, instead of `-g`

`-DCMAKE_CXX_FLAGS_DEBUG=""` sets it to the string containing two double quotes ...

File: performance/benchmarks/benchmark_spherical_representations.py [hunk 2]
```diff
@@ -1,104 +1,109 @@
...
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this software; see the file LICENSE. If not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
 
 # This file should be executed from the building directory
-# quick solution for loading 
-import sys, os
-sys.path.insert(0, os.getcwd())
```
Comment (agoscinski) [inline performance/benchmarks/benchmark_spherical_representations.py]: Hey you mentioned this today I think, but I did not get why this is not needed anymore?

Comment (Luthaf) [reply performance/benchmarks/benchmark_spherical_representations.py]: We set PYTHONPATH instead: https://github.com/cosmo-epfl/librascal/blob/e151317f6461fb440739d4a664818cf654c249ff/performance/benchmarks/CMakeLists.txt#L28-L30
